weblate/api/serializers.py

```diff
@@ -923,6 +924,37 @@ class Meta:
         )
 
 
+class LabelsSerializer(serializers.RelatedField):
+    class Meta:
+        model = Label
+
+    def get_queryset(self):
+        """
+        List of available labels for an unit.
+
+        The queryset argument is only ever required for writable relationship field,
+        in which case it is used for performing the model instance lookup, that maps
+        from the primitive user input, into a model instance.
+        """
+        unit = self.parent.parent.instance
+        project = unit.translation.component.project
+        return project.label_set.all()
```

Comment:
This doesn't seem to be used (not covered by tests), can you please look into that?